### PR TITLE
Bio.Graphics.GenomeDiagram._Diagram: increased test coverage

### PR DESCRIPTION
**Pull request by carlosp420**
from 82% to 98%.

Also reformatted the docstrings for redendering in epydoc or Sphinx.

### CODE REVIEW

File: Tests/test_GenomeDiagram.py [hunk 1]
```diff
@@ -70,21 +64,20 @@ def fill_and_border(base_color, alpha=0.5):
 
 
 def apply_to_window(sequence, window_size, function, step=None):
-    """ apply_to_window(sequence, window_size, function) -> [(int, float),(int, float),...]
-
-        o sequence      Bio.Seq.Seq object
-
-        o window_size   Int describing the length of sequence to consider
+    """Returns a list of (position, value) tuples for fragments of the passed
```
Comment (peterjc) [inline Tests/test_GenomeDiagram.py]: Minor: Try to start the docstring with a single line description then a blank line as per PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

**Comment by peterjc** [comment]
This looks great Carlos :)

The pypy failure on TravisCI looks unrelated, and was probably a stochastic failure - I have clicked on the re-run button.

Any comments @widdowquinn or can I go ahead and merge this?

**Comment by carlosp420** [comment]
doh!, fixed now.

**Comment by widdowquinn** [comment]
I don't see any problems - thanks for doing this Carlos, it's much appreciated!

**Comment by carlosp420** [comment]
:wink:

**Pull request merged**